app/views/content/help-and-advice.md

```diff
@@ -11,22 +11,27 @@ calls_to_action:
     name: chat
     arguments: {}
 ---
+## Get an adviser
 
-## Get a teacher training adviser
+Advisers provide free, one-to-one support by phone, text or email. They're all experienced, former teachers and able to answer any questions you have.
 
-Teacher training advisers provide free, one-to-one support by phone, text or email wherever you are in your journey into teaching. They are experienced, former teachers and able to answer any questions you have. You will have your own dedicated adviser to support you at every step as you make important decisions about your future.
+You will have your own dedicated adviser to support you.
 
-They can help you to:
+### Teacher training advisers
```

Comment:
I'm not sure whether this is a good thing, to send people from Help and advice to the TTA page and the ETA page rather than straight to the funnel - feels like an extra step that won't necessarily add a lot?

This might be a moot point if you decide to change the Help and advice page into a Help category instead though!